packages/datagrid/src/components/default-header-renderer/header-grid.scss

```diff
@@ -0,0 +1,26 @@
+.td-header-component {
+	border-right: #e6e6e6 1px solid;
+
+	> button {
```

Comment:
Can u make a more specific selector than button ?

Reply:
it is not necessary.